tests/failpoints/cases/test_replica_stale_read.rs

```diff
@@ -116,7 +116,7 @@ fn test_stale_read_1pc_flow_replicate() {
     );
     let read_ts = get_tso(&pd_client);
     // wait for advance_resolved_ts.
-    sleep_ms(200);
+    sleep_ms(1000);
```

Comment:
Why the change?

Reply:
To stable the test.

Reply:
How the change can affect the test case?

Reply:
I add 200ms delay in resolved-ts advance to prevent frequent notify

Reply:
Can it be speed up for tests? Otherwise I'm afraid the test time can be way longer.